app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_, file_to_move, path_to_move = command.split()
+        if command_ == "mv" and file_to_move != path_to_move.split("/")[-1]:
+            place_to_move = path_to_move.split("/")[:-1]
+            initial_dir = os.getcwd()
+            for folder in place_to_move:    # Creating folder/folders
+                os.mkdir(folder)
+                os.chdir(f"{os.getcwd()}\\{folder}")
+            os.chdir(initial_dir)   # return to original directory
```

Comment:
Instead of manually creating folders one by one, you can achieve the same result with a single line by using  `os.makedirs with exist_ok=True`